Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
I disagree, politely. I think for upstream that behaviour is perfectly expected & intended. 

Looking at the way upstream has written those files I'm not seeing anything that is behaving in any unexpected way, and beyond that I think this is a perfectly reasonable place to have an exception to the rule because _theoretically_ we never _needed_ to install `npm` as we were doing; we just didn't have a reason to change a stable system before now.